### ISSUE TITLE
Unclear instructions in the COCO API installation

### ISSUE THREAD

**Issue by aklingam7** [sglvladi/TensorFlowObjectDetectionTutorial#47]
The note in the [instructions for the COCO API installation](https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/install.html#coco-api-installation) is a little unclear. It isn't clear where the config file is supposed to be and what the eval_config message is. I spent some time on that step and wasn't able to get anywhere, so I skipped it. The model training went alright, but the evaluation wouldn't work.

**Comment by sglvladi** [comment]
@aklingam7 have you followed the instructions under [Evaluating the Model](https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html#evaluating-the-model-optional)?

**Comment by aklingam7** [comment]
@sglvladi I have followed them. Which is why I thought that could have been the problem. I have not skipped any other steps either.

**Comment by sglvladi** [comment]
@aklingam7 Ok, can you please be more specific about the problems you encountered? Did you get an error?

**Comment by aklingam7** [comment]
Yes, I did. I have forgotten the error, though I can try to reproduce it. 

I am not going to need to use the object detection API anytime soon. I just wanted to tell you that some people might get stuck at that step.

I really liked the tutorial btw, thanks a lot :)
